package.json

```diff
@@ -13,7 +13,8 @@
   ],
   "main": "src/_boot.js",
   "scripts": {
-    "start": "node_modules\\.bin\\electron src --nointro",
+    "start-windows": "node_modules\\.bin\\electron src --nointro",
+    "start-linux": "node_modules/.bin/electron src --nointro",
```

Comment:
How about this magic:

```suggestion
    "start": "electron src --nointro",
```

Please test if it works on your end first. It does on mine (Linux).

Reply:
appears to work on my Mac and with my windows machine!